ipet/evaluation/IPETFilter.py

```diff
@@ -518,6 +569,33 @@ def getName(self):
     def getActiveFilters(self):
         return [f for f in self.filters if f.isActive()]
 
+    @staticmethod
+    def setGlobalDataFrameAndIndex(df : pd.DataFrame, index):
+        """Set global data frame and index for filtering, that will be reused by each filter group
+        """
+
+
+        IPETFilterGroup._glbdf_ = df
+        IPETFilterGroup._glbindex_ = index
+
+        IPETFilterGroup._glbinterrows_ = IPETFilterGroup.computeIntersectionRows(df, index)
+
+    @staticmethod
+    def computeIntersectionRows(df : pd.DataFrame, index):
+        """Compute intersection rows of a given data frame and index
+        """
+
+        logging.info("Computing rows for intersection groups")
+
+        dfindex = df.set_index(index).index
+
+        groups = df.groupby(index)
+        instancecount = groups.apply(len).max()
```

Comment:
Is this always the number of instances? Or is this the maximal number of different instances? What happens if the different sets of instances are not subsets of each other?

Reply:
`instancecount` is the maximum frequency, how often *one* instance occurs. This is nice for concatening a run that is split over two test runs on a disjoint set of instances. Your second question is, I guess, what happens if two testruns have been run on disjoint subgroups, with different settings. I guess that this will produce useless tables, but the comparison is also meaningless. :)

Reply:
Ah okay, so this is after the reduction, and counts how often the instance spread over the columns?